python/cudf_polars/cudf_polars/dsl/translate.py

```diff
@@ -87,7 +87,7 @@ def _(
             if predicate is not None
             else None
         )
-    else:
+    else:  # pragma: no cover; CI tests 1.4
```

Comment:
this is fine for now but if the range of polars versions supports becomes wide we might consider having a couple different coverage rc files that specify which lines to ignore and conditionally decide which one to pass based on the polars version

Reply:
Good point.

We can cover the oldest and newest polars via CI testing, but not sure if it's worth to expand the CI matrix to cover versions in the middle as well.